src/control/system/member.go

```diff
@@ -36,13 +36,18 @@ const (
 	MemberStateStopping MemberState = 0x0010
 	// MemberStateStopped indicates process has been stopped.
 	MemberStateStopped MemberState = 0x0020
-	// MemberStateEvicted indicates rank has been evicted from DAOS system.
+	// MemberStateEvicted indicates rank has been automatically evicted from DAOS system.
 	MemberStateEvicted MemberState = 0x0040
 	// MemberStateErrored indicates the process stopped with errors.
 	MemberStateErrored MemberState = 0x0080
 	// MemberStateUnresponsive indicates the process is not responding.
 	MemberStateUnresponsive MemberState = 0x0100
+	// MemberStateExcluded indicates that the rank has been administratively excluded.
+	MemberStateExcluded MemberState = 0x0200
```

Comment:
where is this state being used or is it just a placeholder for future?

Reply:
Placeholder for now. After some discussion with @liw we decided that there should be a state for ranks that are manually marked out of the system map. There will be some dmg tooling updates required to support this.